project/api/tests.py

```diff
@@ -210,53 +211,40 @@ def test_api_can_create_nation(self):
         self.assertEqual(Nation.objects.count(), 3)
         self.assertEqual(Nation.objects.get(pk=3).name, "Created Test Nation")
 
```

Comment:
I reordered the tests so we can in future break this big `APITest` into `TerritoryAPITest`, `NationAPITest`, etc.

Reply:
Not sure about python best practices, but would separating those tests (as well as models) into separate files be a good idea?

Reply:
Yes, I was meaning to do this after Ata's PR